libs/openchallenges/home/src/lib/challenge-host-list/challenge-host-list.component.html

```diff
@@ -4,14 +4,12 @@
       <h3 class="section-title">Discover Organizations</h3>
       It takes a village to run a challenge - see who has been contributing their time and effort.
     </div>
-    <div class="row">
-      <div class="card-group">
-        <openchallenges-organization-card
-          *ngFor="let organizationCard of organizationCards$ | async"
-          [organizationCard]="organizationCard"
-        >
-        </openchallenges-organization-card>
-      </div>
+    <div class="card-group">
+      <openchallenges-organization-card
```

Comment:
Should we add a paginator to this page?

For challenges with a long list of contributors, e.g. BraTS 2023.

Reply:
Good point! The paginator could be added here, but it will require more work, especially if the param in the url is also needed to reflect the pagination of contributors.

As the profile page is going to be transformed to single page, using the same approach as "Stargazers" to have a "Show More" button seems easier to me. Brats has 35 total, which is not too bad. 

Anyways, hope there's no challenge has tons of contributors before we address it in a different pr XD.